src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/generic-whitelist

```diff
@@ -1,4 +1,13 @@
-# groovy-cps
+# groovy-cps - not needed from workflow-cps 2.33+
+staticMethod com.cloudbees.groovy.cps.CpsDefaultGroovyMethods $addEntry__java_util_Map__java_lang_Object java.util.Map java.lang.Object
```

Comment:
Why are we _adding_ to this list, if 2.32- had only `each`, and 2.33+ looks at the natural method signature? I am not following.